src/class-rest.php

```diff
@@ -93,11 +93,34 @@ public function get_callback( array $object ): array {
 		$post_id = $object['ID'] ?? $object['id'];
 		$options = $this->parsely->get_options();
 		$post    = WP_Post::get_instance( $post_id );
-		$meta    = $this->parsely->construct_parsely_metadata( $options, $post );
 
-		return array(
+		$response = array(
 			'version' => self::REST_VERSION,
-			'meta'    => $meta,
+			'meta'    => $this->parsely->construct_parsely_metadata( $options, $post ),
 		);
+
+		/**
+		 * Filter whether REST API support in rendered string format is enabled or not.
+		 *
+		 * @since 3.1.0
+		 *
+		 * @param bool $enabled True if enabled, false if not.
+		 */
+		if ( apply_filters( 'wp_parsely_enable_rest_rendered_support', true ) ) {
```

Comment:
Since we have already inflated the `WP_Post` instance, it'd be nice to include in the available params to filtering functions:

```suggestion
		if ( apply_filters( 'wp_parsely_enable_rest_rendered_support', true, $post ) ) {
```